test/unittests/ngsi10/UpdateContextRequest_test.cpp

```diff
@@ -70,7 +70,7 @@ TEST(UpdateContextRequest, ok_json)
    //
    UpdateContextRequest*  upcrP = &reqData.upcr.res;
 
-   upcrP->present(""); // No output
+//   upcrP->present(""); // No output
```

Comment:
It seems a leftover. I understand this line should be removed.

Reply:
fixed in 96e667a